benchmark/assert/deepequal-prims-and-objs-big-loop.js

```diff
@@ -16,33 +17,54 @@ const primValues = {
 const bench = common.createBenchmark(main, {
   prim: Object.keys(primValues),
   n: [1e6],
-  method: ['strict', 'nonstrict']
+  method: [
+    'deepEqual',
+    'deepStrictEqual',
+    'notDeepEqual',
+    'notDeepStrictEqual'
+  ]
 });
 
 function main(conf) {
   const prim = primValues[conf.prim];
   const n = +conf.n;
   const actual = prim;
   const expected = prim;
+  const expectedWrong = 'b';
   var i;
 
   // Creates new array to avoid loop invariant code motion
   switch (conf.method) {
-    case 'strict':
+    case 'deepEqual':
       bench.start();
       for (i = 0; i < n; ++i) {
         // eslint-disable-next-line no-restricted-properties
         assert.deepEqual([actual], [expected]);
```

Comment:
I think I prefer to stick to the way it is as that way it's simpler to add more tests for the same function with different inputs.